### DISCUSSION TITLE
Why is it that you can't move on after about 1000 training sessions

### DISCUSSION THREAD

**Discussion by chuckchang6** [open-mmlab/mmclassification#1611] [Q&A] [answered]
<img width="1014" alt="image" src="https://github.com/open-mmlab/mmpretrain/assets/108621318/120fd70f-1bc5-494d-b239-0a7848692634">
<img width="1013" alt="image" src="https://github.com/open-mmlab/mmpretrain/assets/108621318/57a25b76-d1e8-494a-aec1-f7db28a0ce4c">

**Comment by Ezra-Yu** [comment, marked as answer]
CUDA out of memory! Maybe another work takes up the GPU.

**Reply by chuckchang6** [reply]
thanks！i try to find out this
